pootle/core/forms.py

```diff
@@ -154,23 +155,162 @@ def clean(self):
 
 
 class PaginatingForm(forms.Form):
-    page = forms.IntegerField(
-        initial=1, min_value=1, max_value=100)
+    page_field = "page_no"
+    per_page_field = "results_per_page"
+    page_no = forms.IntegerField(
+        required=False,
+        initial=1,
+        min_value=1,
+        max_value=100)
     results_per_page = forms.IntegerField(
+        required=False,
         initial=10,
-        min_value=0,
+        min_value=10,
         max_value=100,
         widget=forms.NumberInput(attrs=dict(step=10)))
 
 
 class FormWithActionsMixin(forms.Form):
-    pass
-
-
-class FormtableForm(PaginatingForm, FormWithActionsMixin):
     action_choices = ()
+    action_field = "actions"
+    comment_field = "comment"
+    select_all_field = "select_all"
     actions = forms.ChoiceField(
         required=False,
         label=_("With selected"),
         widget=forms.Select(attrs={'class': 'js-select2'}),
-        choices=action_choices)
+        choices=(
+            ("", "----"),
+            ("reject", "Reject"),
+            ("accept", "Accept")))
+    comment = forms.CharField(
+        label=_("add comment"),
```

Comment:
in this case it wants to be lower in the actual view - but ill update and make it lower in the template